apps/src/code-studio/pd/professional_learning_landing/SelfPacedProgressTable.jsx

```diff
@@ -0,0 +1,131 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import {studio} from '@cdo/apps/lib/util/urlHelpers';
+import Link from '@cdo/apps/componentLibrary/link';
+import {LinkButton} from '@cdo/apps/componentLibrary/button';
+import {
+  BodyThreeText,
+  BodyFourText,
+} from '@cdo/apps/componentLibrary/typography';
+import styles from './selfPacedProgressTable.module.scss';
+// Generic table styles that are shared with LandingPage.jsx
+import './tableStyles.scss';
+
+const ProgressBar = ({percentCompleted}) => {
```

Comment:
Do y'all think it's worth moving this into it's own component so it can be shared?